midifile.c

```diff
@@ -17,8 +17,52 @@
 #include "plugout.h"
 #include "midifile.h"
 
-#define TRACK_LENGTH_OFFSET 18
-#define TRACK_START_OFFSET 22
+/*
+ * The hardware emulation runs at 4194304 Hz and provides an exact
+ * cycle counter.  Because ~4 MHz are outside of the possible MIDI
+ * time resolution, we introduce an arbitrary lower resolution of 256
+ * Hz by shifting the cycle counter 14 bits to the right.
+ *
+ * This lower resolution (#FIXME: which needs a nice name) is
+ *
+ *  4194304 Hz / 2^14 == 1/256 Hz == 3906 ms
```

Comment:
== 256 Hz == 3906 µs

Reply:
Or 3.906 ms if we want to avoid the µ :)